v2/tools/generator/internal/codegen/code_generator.go

```diff
@@ -199,6 +199,7 @@ func createAllPipelineStages(
 		pipeline.StripUnreferencedTypeDefinitions(),
 
 		pipeline.RenameProperties(configuration.ObjectModelConfiguration),
+		pipeline.OverrideDescriptions(configuration), // After flatten and rename to make easier to use
```

Comment:
Suggest enforcing this with use of RequiresPrerequisiteStages() when defining the stage.